code/modules/power/singularity/singularity.dm

```diff
@@ -320,6 +326,28 @@
 	consumed_supermatter = TRUE
 	set_light(10)
 
+/obj/singularity/proc/consume_boh(obj/boh)
+	collapsing = TRUE
+	name = "unstable [initial(name)]"
+	desc = "[initial(desc)] It seems to be collapsing in on itself."
+	visible_message(
+		message = span_danger("As [src] consumes [boh], it begins to collapse in on itself!"),
+		blind_message = span_hear("You hear aggressive crackling!"),
+		vision_distance = 15,
+	)
+	var/sound/collapse = sound('sound/effects/clockcult_gateway_disrupted.ogg')
+	collapse.pitch = -1
+	collapse.frequency = 0.5
+	playsound(loc, collapse, 200, vary = TRUE, extrarange = 3, falloff_exponent = 1, pressure_affected = FALSE, ignore_walls = TRUE, falloff_distance = 7)
+	addtimer(CALLBACK(src, PROC_REF(consume_boh_sfx)), 4 SECONDS)
+	animate(src, time = 4.5 SECONDS, transform = matrix(transform).Scale(0.25), flags = ANIMATION_PARALLEL, easing = ELASTIC_EASING)
+	animate(time = 0.5 SECONDS, alpha = 0)
+	QDEL_IN(src, 4.1 SECONDS)
```

Comment:
don't you mean 4.6

Reply:
or like idk 4.3

Reply:
Some some reason probably related to animates being client side making the `qdel` timer equal to or longer than the animate timers results in a noticeably long wait between the animation finishing and the qdel happening. 

So I just tweaked the timers down until they looked good (which is where they are now)